Codegen/src/GeneratorHelper.cs

```diff
@@ -99,5 +101,35 @@ public static string ComputeIndexHashName(string qualified_name, Node sourceNode
             return hashName;
         }
 
+        /// <summary>
+        /// Return the list of indented lines of copy node declaration
+        /// </summary>
+        /// <param name="copyNode">Node copy</param>
+        /// <returns>The list of ITextLines.</returns>
+        public static List<ITextLine> GetCopyLines(Node copyNode)
```

Comment:
- can be refactored into a deferred enumeration method (and also an extension method if you like)
- I think the input parameter can be more strictly typed as `LinearCopyNode`

Reply:
Now, it's a deferred enumeration method of the class LinearCopyNode.
https://github.com/TypeCobolTeam/TypeCobol/pull/1648/commits/31338d478e73bd6e55f1aca16dd21ebabd1e9eff